cli/version.rs

```diff
@@ -9,6 +9,8 @@ const TYPESCRIPT: &str = env!("TS_VERSION");
 const CARGO_PKG_VERSION: &str = env!("CARGO_PKG_VERSION");
 // TODO(bartlomieju): ideally we could remove this const.
 const IS_CANARY: bool = option_env!("DENO_CANARY").is_some();
+// TODO(bartlomieju): this is temporary, to allow Homebrew to cut RC releases as well
+const IS_RC: bool = option_env!("DENO_CANARY").is_some();
```

Comment:
It's ok to be IS_CANARY and IS_RC?

Reply:
Yeah, I'd think this has to be `DENO_RC`. Otherwise, there's no difference between `IS_CANARY` and `IS_RC`.

Reply:
Yup, a mistake here